src/__tests__/ActionMenu.test.tsx

```diff
@@ -113,10 +114,10 @@ describe('ActionMenu', () => {
   it('should be able to select an Item with selectionVariant', async () => {
     const component = HTMLRender(
       <ThemeProvider theme={theme}>
-        <SingleSelection />
+        <SingleSelect />
       </ThemeProvider>,
     )
-    const button = component.getByLabelText('Field type')
+    const button = component.getByRole('button')
```

Comment:
Can someone help me improve this test? Im not sure how to select this button as I've made the button label dynamic.

Reply:
You could use a regex
```suggestion
    const button = component.getByLabelText(/^options/i)
```